src/components/SingleNotification.tsx

```diff
@@ -51,26 +77,40 @@ const SingleNotification = ({
           <div>
             <div className="d-flex justify-content-start">
               <div className="pb-3 pe-1">
-                <StyledImage roundedCircle fluid src={userlogo} />
+                {likes.docs.map((item) => (
+                  <span key={item._id}>
+                    <StyledImage
+                      className="me-2"
+                      roundedCircle
+                      fluid
+                      src={
+                        item.user?.profile?.avatar
+                          ? item.user.profile.avatar
+                          : "https://abs.twimg.com/sticky/default_profile_images/default_profile_400x400.png"
+                      }
+                      key={item._id}
+                    />
```

Comment:
Replace StyledImage on Avatar.tsx component

Reply:
Thanks for the tip. Done.
<img width="643" alt="image" src="https://user-images.githubusercontent.com/103769012/193032922-7486baa5-44e4-44aa-b883-e39b7ec76cee.png">